src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -45,7 +45,7 @@ public sealed class NpgsqlDataReader : DbDataReader
         /// </summary>
         CommandBehavior _behavior;
 
-        Task _sendTask = default!;
+        //Task _sendTask = default!;
```

Comment:
Commented?

Reply:
Brought back the sendTask functionality.